src/tic_tac_toe/board.clj

```diff
@@ -16,39 +16,35 @@
     false
     (apply = (row board))))
 
-(def winning-positions-four-by-four
-                        [[0 1 2 3]
-                        [4 5 6 7] 
-                        [8 9 10 11] 
-                        [12 13 14 15]
-                        [0 4 8 12]
-                        [1 5 9 13]
-                        [2 6 10 14]
-                        [3 7 11 15]
-                        [0 5 10 15]
-                        [3 6 9 3]
-                         ])
-
-(def winning-positions [[0 1 2] 
-                        [3 4 5] 
-                        [6 7 8]
-                        [1 4 7]
-                        [2 5 8]
-                        [0 3 6]
-                        [0 4 8]
-                        [2 4 6]])
+(defn- gen-diagonals-left [board]
```

Comment:
not sure on what the `gen` prefix adds here. I would not abbreviate words when doing naming as it makes it a little ambigious as to what "gen" means.

If they are just the diagonal positions, I would just cal them `diagonals-left`, `columns`, etc.